features/step_definitions/message_steps.ts

```diff
@@ -23,10 +23,15 @@ type StringifiedStatus =
   | 'AMBIGUOUS'
   | 'FAILED'
 
-const { Status } = messages.TestResult
+const { Status } = messages.TestStepFinished.TestStepResult
+
+const ENCODING_MAP: { [key: string]: messages.Attachment.ContentEncoding } = {
+  IDENTITY: messages.Attachment.ContentEncoding.IDENTITY,
+  BASE64: messages.Attachment.ContentEncoding.BASE64,
+}
```

Comment:
I don't think this is needed. I think we could replace its usage with:

```typescript
ENCODING_MAP[x['MEDIA ENCODING']]
// becomes
messages.Attachment.ContentEncoding[x['MEDIA ENCODING']]
```

Reply:
Will try but don’t think so - that enum will compile down to just the ordinals, not the strings

Reply:
(it does just compile down to the ordinals sadly)